packages/accounts/src/light-account/account.ts

```diff
@@ -20,10 +25,51 @@ import { LightAccountFactoryAbi } from "./abis/LightAccountFactoryAbi.js";
 export class LightSmartContractAccount<
   TTransport extends Transport | FallbackTransport = Transport
 > extends SimpleSmartContractAccount<TTransport> {
-  override async signTypedData(params: SignTypedDataParams): Promise<Hash> {
-    return this.owner.signTypedData(params);
+  override async signMessage(msg: string | Uint8Array): Promise<`0x${string}`> {
+    return this.signWith1271Wrapper(
+      hashMessage(
+        typeof msg === "string" && !isBytes(msg)
+          ? msg
+          : {
+              raw: msg,
+            }
+      )
+    );
   }
 
+  override async signTypedData(
+    params: SignTypedDataParams
+  ): Promise<`0x${string}`> {
+    return this.signWith1271Wrapper(hashTypedData(params));
+  }
+
+  override async signUserOperationHash(
+    uoHash: `0x${string}`
+  ): Promise<`0x${string}`> {
+    return this.owner.signMessage(uoHash);
+  }
+
+  private signWith1271Wrapper = async (msg: Hash): Promise<`0x${string}`> => {
+    return this.owner.signTypedData({
+      domain: {
```

Comment:
@howydev can you confirm these values are correctly set for light account? for msca we can use the `eip712domain` view function, but that's not here

Reply:
bump @howydev